src/Akka.Streams.Kafka.Tests/Integration/PlainSinkIntegrationTests.cs

```diff
@@ -88,5 +91,76 @@ public async Task PlainSink_should_fail_stage_if_broker_unavailable()
             probe.ExpectSubscription();
             probe.OnError(new KafkaException(ErrorCode.Local_Transport));
         }
+        
+        [Fact]
+        public async Task PlainSink_should_resume_on_deserialization_errors()
```

Comment:
Should make it clear that we're testing a custom decider here, not the built-in one.

Reply:
Done